packages/terra-worklist-data-grid/src/subcomponents/RowSelectionCell.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import '../_elementPolyfill';
 import { injectIntl } from 'react-intl';
+
 import Cell from './Cell';
+import '../_elementPolyfill';
```

Comment:
This polyfill is not necessary.

Reply:
It does not appear that this change is necessary.  If we want to make a change here, we should remove the polyfill as it is not used.